game/maps/megaship/megaship_aeration.rpy

```diff
@@ -71,7 +71,7 @@ label loop_megaship_aeration:
                             call dialog_nosense from _call_dialog_nosense_29
                         else:
                             call dialog_closed from _call_dialog_closed_31
-                        m "How to cut this metal grid? {w=3} {nw}"
+                        m "How do you want me to cut this metal grid? {w=3} {nw}"
```

Comment:
In the context, I'm not sure about this one. Maybe something like this could be better?: "How could I cut this metal grid?"

Reply:
thanks!